omnibus/files/private-chef-cookbooks/private-chef/recipes/bootstrap.rb

```diff
@@ -1,70 +1,59 @@
 #
-# Author:: Adam Jacob (<adam@opscode.com>)
-# Copyright:: Copyright (c) 2011 Opscode, Inc.
+# Author:: Adam Jacob <adam@chef.io>
+# Copyright:: Copyright (c) 2011-2015 Chef Software, Inc.
 #
 # All Rights Reserved
 #
 
-require 'securerandom'
-
 # Enterprise Addon Install
-# On a new install, download and install all of the Chef
-# addons from package repositories. Only perform this
-# action during a fresh install, and not during an upgrade.
+include_recipe "private-chef::opscode-erchef"
+
+# When we're running a new standalone install and are configured
+# to install addons, do so now from the package repositories.
 if (!OmnibusHelper.has_been_bootstrapped? &&
     node['private_chef']['topology'] == "standalone" &&
     node['private_chef']['addons']['install'])
   include_recipe "private-chef::add_ons_wrapper"
 end
 
-opscode_test_dir = "/opt/opscode/embedded/service/chef-server-bootstrap"
-opscode_test_config_dir = "/opt/opscode/embedded/service/chef-server-bootstrap/bootstrapper-config"
-
-template File.join(opscode_test_config_dir, "config.rb") do
-  source "bootstrap-config.rb.erb"
-  owner "root"
-  group "root"
-  mode "0600"
+pivotal_key_path = "/etc/opscode/pivotal.pem"
+if File.exists?(pivotal_key_path)
+  pivotal_key = OpenSSL::PKey::RSA.new(pivotal_key_path)
+else
+  pivotal_key = OpenSSL::PKey::RSA.generate(2048)
```

Comment:
I'd been thinking the same as I went through bootstrap.  There is no reason not to add them as multiple pivotal keys, and it simplifies things for the config picture.